src/clr-angular/forms/styles/_containers.clarity.scss

```diff
@@ -28,7 +28,7 @@
     color: $clr-forms-label-color;
     font-size: $clr-forms-label-font-size;
     font-weight: $clr-forms-label-font-weight;
-    line-height: $clr-forms-baseline * 3;
+    line-height: $clr-forms-baseline * 4;
```

Comment:
The fix here is different, because making a larger line-height causes issues when the label is longer than its space and wraps. We want to have 18px line height for those cases, so the fix I believe would be putting a `margin-top: 0.125rem` on the label only when it is in horizontal or compact mode. This is actually fixed in compact mode further down.